src/app/components/JumpTo/index.tsx

```diff
@@ -22,7 +22,6 @@ const JumpTo = ({ jumpToHeadings }: JumpToProps) => {
   const eventTrackingData: EventTrackingMetadata = {
     componentName: 'jumpto',
     optimizely,
-    optimizelyEventName: 'jumpto',
```

Comment:
What does this mean if this is removed, as I don't see this event name anywhere else

Reply:
The prop isn't needed and `optimizelyMetricNameOverride` has taken its place. The original was added by me as I thought it would achieve the behaviour that this PR introduces, but it didn't.

Reply:
The JumpTo component itself doesn't need to have its metric name overridden, as it will use the `component_views` and `component_clicks` metric names.